index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/styles.css">
+    <script src="https://kit.fontawesome.com/cebf050324.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <div class="background">
+      <nav>
+        <ul class="list-format font-color">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
```

Comment:
- hey, in the mobile version, theses links should not be visible. let's add the hamburger icon and make these links not visible for the mobile version. see the Figma template to update this change.